spec/api-notification-spec.js

```diff
@@ -66,10 +70,10 @@ describe('Notification module', () => {
       ]
     })
 
-    assert.equal(n.actions[0].type, 'button')
-    assert.equal(n.actions[0].text, '1')
-    assert.equal(n.actions[1].type, 'button')
-    assert.equal(n.actions[1].text, '2')
+    expect(n.actions[0].type).to.equal('button')
```

Comment:
Shouldn't we check that `n.actions` is an array of the expected size first?

Reply:
done